src/components/App.tsx

```diff
@@ -229,6 +246,14 @@ export const App = observer(() => {
           />
         </>
       }
+      <div
+        className="visually-hidden"
+        role="alert"
+        aria-live="assertive"
+        aria-atomic="true"
+      >
+        {ariaLiveText}
+      </div>
```

Comment:
I'm not sure where it makes the most sense to put it, but I think a comment or comments about why this exists and how it works would be helpful for other developers. Maybe a brief comment here and then another in the new aria-live-context.tsx file?